testing/test_select.py

```diff
@@ -38,3 +39,48 @@ def test_select_item_select(select):
     with pytest.raises(SelectItemNotFound):
         select.fill("Non existing item")
     assert not select.is_open
+
+
+@pytest.fixture
+def checkbox_select(browser):
+    return CheckboxSelect(
+        browser,
+        locator=(
+            './/div[@id="ws-react-c-select-checkbox-input"]'
+            '//div[contains(@class, "pf-c-select")]'
+        ),
+    )
+
+
+def test_checkbox_select_is_displayed(checkbox_select):
+    assert checkbox_select.is_displayed
+
+
+def test_checkbox_select_items(checkbox_select):
+    assert set(checkbox_select.items) == {"Active", "Cancelled", "Paused", "Warning", "Restarted"}
+    assert checkbox_select.has_item("Active")
+    assert not checkbox_select.has_item("Non existing item")
+    assert checkbox_select.item_enabled("Paused")
+
+
+def test_checkbox_select_open(checkbox_select):
+    assert not checkbox_select.is_open
+    checkbox_select.open()
+    assert checkbox_select.is_open
+    checkbox_select.close()
+    assert not checkbox_select.is_open
+
+
+def test_checkbox_select_item_checkbox_select(checkbox_select):
+    checkbox_select.fill({"Restarted": True})
+    assert checkbox_select.read() == {
+        "Active": False,
+        "Cancelled": False,
+        "Paused": False,
+        "Warning": False,
+        "Restarted": True,
+    }
```

Comment:
Can we just test that a `fill` works as well.

So run a `fill` to select some items (set them to True) -- and then run a read and see if the result matches the fill request